### PR TITLE
Create Agenda model, import task for Philly city council agendas

### PR DESCRIPTION
**Pull request by derekeder**
Created a simple Agenda model and rake task to populate it from mongo database 'opengovernment_import'.

There doesn't seem to be much of an established standard for Agendas, so I'm going with the following:

`meeting_date` date and time of meeting
`name` - meeting title (eg City Council, Committee on Public Property and Public Works)
`location` - In Philly's case, this is just a room number, but we should expect to see addresses
`agenda` - URL and full text scan of agenda PDF (when available)
`minutes` - URL and full text scan of minutes PDF (when available)

### CODE REVIEW

File: lib/tasks/city_council.rake [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    #     'url': u'View.ashx?M=A&ID=248602&GUID=2D23A8B0-4FB3-4EF2-A957-C6FD8FF9BDAA', 
+    #     'label': u'Agenda'
+    #   }, 
+    #   u'Minutes': {
+    #     'url': u'View.ashx?M=A&ID=248602&GUID=2D23A8B0-4FB3-4EF2-A957-C6FD8FF9BDAA', 
+    #     'label': u'Minutes'
+    #   }, 
+    # }
+
+    session = Moped::Session.new([ "127.0.0.1:27017" ])
```
Comment (derekeder) [inline lib/tasks/city_council.rake]: I realize this is a sloppy way to get a connection to the database, but wasnt sure what the exact best practice was for connecting to another mongo database we don't have a model for.

**Comment by derekeder** [comment]
### Done
- Model renamed from Agenda to Meeting
- moved import logic to load_from_apis_for_jurisdiction in Meeting model
- Added spec for name, date, municipality

### To do
- determine best way to store unique ID. should include: date, name, agenda PDF url
- figure out the best way to pass yml config to the Moped session in Meeting model

**Comment by walter** [comment]
Am I right in thinking the work that remains on this is that we still need the Agenda and Minutes as embedded models for the Meeting?

**Comment by walter** [comment]
A good example of how embedded models work is person_detail and link models.

**Comment by derekeder** [comment]
@walter all right I think this is ready to get merged in.

**Comment by derekeder** [comment]
@walter ok, variables renamed.

**Comment by walter** [comment]
I think I'm going to merge this for speed of refactoring. Would be a good idea to keep an eye on my changes.

Quick points before I do.
- If we can, we should embeds_one :agenda and embeds_one :minutes (and embedded_in :meeting on the agenda and minutes side)
- I think we can ditch Meeting as prefix for these classes - Agenda and Minutes - if we come across something other than a Meeting that has these, we can refactor our modeling then
- fulltext should be full_text as per Ruby idioms

**Comment by walter** [comment]
Oh, actually one more thing.

I would love it if the class method load_from_apis_for_jurisdiction was spec'd. If you can do that for me, then I'll merge this as that will allow me to refactor with confidence.

**Comment by walter** [comment]
For the load_from_apis_for_jurisdiction spec, check out [FederalLegislator's example](https://github.com/opengovernment/opengovernment-app/blob/master/spec/models/federal_legislator_spec.rb#L94-L99).

You won't need to use :vcr though.

**Comment by derekeder** [comment]
will do!

**Comment by walter** [comment]
Starting refactor. I'll take it from here. Still need to make it so that mocked version of scraped_local_gov allows our test to pass. Will work on it.

**Comment by walter** [comment]
Hey @evz and @derekeder, can you not work off datamade fork at the moment. It's confusing this pull request (we should have used a topic branch).

I've added @evz as a collaborator. You can do a direct clone instead of the fork.

**Comment by evz** [comment]
@walter @derekeder Whoops, sorry about that. Do you want me to revert that commit? It was just a few observations that I made while setting this up for development lat night.

**Comment by walter** [comment]
I think if you leave it in datamade's fork for now, that will be fine. I'll merge separately later.

**Comment by walter** [comment]
Finished refactor, etc.

Next up will be adding display of agendas for Philly pages. Will handle that in separate issue.

**Pull request closed without merging**